client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -51,7 +51,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
     const [order, setOrder] = useState<Order>(SortOrder.asc);
     const [orderBy, setOrderBy] = useState<string>(defaultOrderBy);
     const [allStatuses, setAllStatuses] = useState<InvestigationMainStatus[]>([]);
-    const [allSubStatuses, setAllSubStatuses] = useState<InvestigationSubStatus[]>([]);
```

Comment:
Just a question, what was InvestigationSubSatuse's purpose in the first place? how is it different from SubStatus?

Reply:
it was just the display name without any additional info